sqlite/backend.rs

```diff
@@ -312,10 +318,32 @@ impl SqliteBackend {
               .execute(params![mutation.key])?;
             assert!(changed == 0 || changed == 1)
           }
-          MutationKind::Sum(operand) => {
-            mutate_le64(tx, &mutation.key, "sum", operand, version, |a, b| {
-              a.wrapping_add(b)
-            })?;
+          MutationKind::Sum {
+            value: operand,
+            min_v8,
+            max_v8,
+            clamp,
+          } => {
+            if matches!(operand, KvValue::U64(_)) {
+              mutate_le64(
+                tx,
+                &mutation.key,
+                "sum",
+                operand,
+                version,
+                |a, b| a.wrapping_add(b),
+              )?;
```

Comment:
How can this code path be triggered from JS? By using `Deno.AtomicOperation.mutate()` method, and passing `KvU64` value?

Reply:
Yes